### PR TITLE
boring kyber bench fix

### PR DESCRIPTION
**Pull request by franziskuskiefer**

### CODE REVIEW

File: benches/boringssl/kyber768.cxx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#include <string>
+#include <cstring>
```
Comment (xvzcf) [inline benches/boringssl/kyber768.cxx:1]: We could just remove this line altogether

Comment (franziskuskiefer) [reply benches/boringssl/kyber768.cxx]: sure, if it's not used ... let's see what the CI says.
Also there's something off on the CI. It works on main but fails here. I have to look into that.

Comment (franziskuskiefer) [reply benches/boringssl/kyber768.cxx:1]: It's needed, you use memset

**Review by xvzcf — APPROVED** [review]

File: benches/boringssl/kyber768.cxx (hunk 1 above)
Comment (xvzcf) [reply benches/boringssl/kyber768.cxx:1]: Strange that I can build locally without the header, but oh well

**Pull request merged**